src/Oracles.sol

```diff
@@ -34,13 +34,23 @@ contract ForwardingProofMockBridge is IBridgeContract {
     }
 
     function _execute(string calldata _sourceChain_, string calldata sourceAddress_, bytes calldata payload_) external override {
-       require(StringsEqual(_sourceChain_, "FIL"), "Only FIL proofs supported");   
+       require(StringsEqual(_sourceChain_, "filecoin-2") && block.chainid == 314159, "Only FIL proofs supported");   
```

Comment:
This is redundant, I don't see why to include it here.

Reply:
I was thinking if someone mistakingly puts MockBridge address instead of Axelar bridge but yeah, that transaction would eventually fail as gateway is not configured, removing it.